charts/etcd/templates/configmap-etcd-bootstrap.yaml

```
@@ -19,14 +19,21 @@ metadata:
     uid: {{ .Values.uid }}
 data:
   bootstrap.sh: |-
-    #!/bin/sh
+    #!/bin/sh -e
```

```suggestion
    #!/bin/sh
```
I tried this with `-e`. The VALIDATIONMARKER is not created. Lets remove `-e` as it will affect the validation flow.
```
#!/bin/sh -e

trap_and_propagate() {
        PID=$1
        shift
        for sig in "$@" ; do
            trap "kill -$sig $PID" "$sig"
        done
}

rm -rf $VALIDATION_MARKER
etcd &
ETCDPID=$!
trap_and_propagate $ETCDPID INT TERM
wait $ETCDPID
RET=$?
echo $RET > VALIDATION_MARKER
exit $RET
```